Assets/Scripts/System/AddComponentSystem.cs

```diff
@@ -0,0 +1,44 @@
+using System;
+using System.Collections.Generic;
+using Assets.Scripts.SceneState;
+using Assets.Scripts.Utility;
+using UnityEditor;
+using UnityEngine;
+using Zenject;
+
+namespace Assets.Scripts.System
+{
+    public class AddComponentSystem
+    {
+        // Dependencies
+        private CommandSystem commandSystem;
+
+        [Inject]
+        private void Construct(CommandSystem commandSystem)
+        {
+            this.commandSystem = commandSystem;
+        }
+
+        public List<DclComponent.ComponentDefinition> GetAvailableComponents()
```

Comment:
Is there a reason this is a method and not a `static` and/or `cont` field?

I sugest something like:
`public static List<T> availableComponents`
But I also know there are some difficulties with initializing `const` stuff in C#.